llvm/test/Transforms/PGOProfile/loop3.ll

```diff
@@ -0,0 +1,78 @@
+; RUN: opt %s -passes=pgo-instr-gen -pgo-instrument-loop-entries=false -S | FileCheck %s --check-prefixes=GEN,NOTLOOPENTRIES
+; RUN: llvm-profdata merge %S/Inputs/loop3.proftext -o %t.profdata
```

Comment:
Instead of creating new files in the `Inputs` directory, lets use `split-file` to inline the `.proftext` files into this test.

https://llvm.org/docs/TestingGuide.html#extra-files

Reply:
Also, it might be cleaner to split this into two separate tests: `PGOProfile/loop-entry-gen.ll` and `PGOProfile/loop-entry-use.ll`.